config/00-main.yaml

```diff
@@ -5,13 +5,13 @@ ores:
     application_root: ""
     url_prefix: ""
   data_paths:
-    nltk: /srv/ores/config/submodules/wheels/nltk/
```

Comment:
You can use the config merging capability to modify the config values.  E.g. if you save the following next to `00-main.yaml` in a file named `50-deployment.yaml` it will make the changes you have made.

```
ores:
  data_paths:
    nltk: /srv/ores/deploy/submodules/wheels/nltk/

score_caches:
  ores_redis:
    host: deployment-ores-redis

score_processors:
  ores_celery:
    BROKER_URL: redis://deployment-ores-redis:6379
    CELERY_RESULT_BACKEND: redis://deployment-ores-redis:6379